### PR TITLE
initial commit of plot_convergence

### PR DESCRIPTION
**Pull request by pgkirsch**
To try and help debug/understand my SP solving woes, I wrote a quick script (`plot_convergence`) to visualize the convergence of the decision variables. This is another tool where I don't know if it belongs in gpkit or not, but I don't know someone else might find this useful too.

Example usage:
```python
In [1]: %run tail.py

In [2]: import plot_convergence as pc

In [3]: pc.plot_convergence(m)
```
screen shot 2015-10-29 at 01 46 29 am

In case anyone is wondering, this was motivated by wanting to know what is going on when the solver exhausts the 52 solutions without converging. This example shows how the solution is oscillating.

screen shot 2015-10-29 at 01 47 44 am

The number in parentheses in the maximum value of the DV.
Also matplotlib legends are the absolute worst.

### CODE REVIEW

**Comment by pgkirsch** [comment]
Obviously if this were to be incorporated into gpkit, I would make this a method of SP or something similar.

**Comment by whoburg** [comment]
this is cool!

**Comment by bqpd** [comment]
Super nifty, definitely tickle my controls brain. The first one looks kinda first-order exponential, while the second keeps kicking into a mode-switch (fesaibility solve) and then flying away...

**Comment by whoburg** [comment]
this is plotting normalized *logs* of variables, I assume?

**Comment by pgkirsch** [comment]
mmmm no, not logs. I just took the values from `m.program.gps[j].result['variables']`.

**Comment by whoburg** [comment]
Some of the variables are ~ 1, and others are ~10^17 (according the the legend). How on earth are they not showing up super close to zero or one, if they're not logs?

**Comment by pgkirsch** [comment]
I'm confused. They are aren't they?

**Comment by whoburg** [comment]
oh! You normalize the variables individually against their maximum values, not as a group. Duh. Everything makes sense.

**Comment by whoburg** [comment]
@bqpd, what do you think of this? Should it potentially get added to `gpkit.interactive`? If so, I'll do an initial review.

**Comment by bqpd** [comment]
Yeah! I don't think it should be normalized by max value, though, because our initial guess is all ones...either a logy plot or normalize by max of solves after the first that weren't feasibility solves?

**Comment by whoburg** [comment]
@pgkirsch, thoughts on @bqpd's question about what the normalization should be?

My initial thought is that I like the idea of a logy plot.

**Comment by pgkirsch** [comment]
Yeah that sounds fine with me. I will change that and put in gpkit.interactive.

**Comment by pgkirsch** [comment]
Ready for another review @whoburg

File: gpkit/interactive/plot_convergence.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    m: Signomial programming model that has already been solved
+    
+    Returns
+    -------
+    Doesn't return anything, but produces a plot
+    """
+
+    keyList = m.program.gps[0].result['variables'].keys()
+
+    newDict = {}
```
Comment (whoburg) [inline gpkit/interactive/plot_convergence.py]: `newDict` should be named more descriptively, if possible.

**Comment by whoburg** [comment]
@pgkirsch, apologies for the delay. #497 is ready for your review. If you like it, merge it into this, and I'm then ready to merge this.

**Pull request merged**